app/backend/src/clj/teet/thk/thk_integration_ion.clj

```diff
@@ -265,3 +266,61 @@
            csv->file
            dump))
   :ok)
+
+
+(def date-fixup-test-event
```

Comment:
Is this leftover from development?

Reply:
It's something to keep around when testing the code again later.